go.mod

```diff
@@ -1,11 +1,18 @@
 module github.com/envoyproxy/xds-relay
 
-go 1.14
+go 1.13
 
 require (
+	github.com/bufbuild/buf v0.7.0 // indirect
 	github.com/envoyproxy/go-control-plane v0.9.4
-	github.com/onsi/ginkgo v1.12.0
-	github.com/onsi/gomega v1.9.0
-	go.uber.org/zap v1.14.0
-	google.golang.org/grpc v1.27.1
+	github.com/envoyproxy/protoc-gen-validate v0.3.0
+	github.com/go-swagger/go-swagger v0.22.0 // indirect
+	github.com/golang/protobuf v1.3.4
+	github.com/grpc-ecosystem/grpc-gateway v1.13.0 // indirect
+	github.com/iancoleman/strcase v0.0.0-20191112232945-16388991a334 // indirect
```

Comment:
Can we remove the indirect packages?

Reply:
actually we can't, these are pulled in by the calls to `go get` in the `generate-api-protos` script.

Reply:
User has to manually commit the changes in this file though right? I would just go in and delete the indirects.

Reply:
cc @twoism, @rodaine here for advice on best Go practices